ui/lib/core/addon/components/alert-banner.hbs

```diff
@@ -30,6 +30,12 @@
             {{yield}}
           </p>
         {{/if}}
+        {{#if @learnLink}}
```

Comment:
Something we've started doing a bit more is adding customizations like this by yielding them. This style seems pretty specific, especially with the icon, what are your thoughts on passing lines 34-37 into the block of `AlertBanner` instead of making a whole new arg?  I also wonder if the new class will even be necessary then 🤔

Reply:
Ohhh! I really like that idea. I'll do that.